package.json

```diff
@@ -18,21 +18,21 @@
   "author": "Team Pathfinder <team-pathfinder@zalando.de>",
   "license": "MIT",
   "devDependencies": {
-    "@types/mocha": "^2.2.44",
-    "@types/node": "^8.0.53",
-    "eslint": "4.7.2",
-    "eslint-config-standard": "^10.2.1",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-node": "^5.2.1",
-    "eslint-plugin-promise": "^3.6.0",
-    "eslint-plugin-standard": "^3.0.1",
-    "jsdoc-to-markdown": "3.0.0",
-    "mocha": "3.5.3",
+    "@types/mocha": "2.2.44",
+    "@types/node": "8.0.54",
+    "eslint": "4.12.1",
```

Comment:
IMO, we do not need to enforce the deps here since we have lock files exactly for that purpose.